src/core/base.js

```diff
@@ -75,6 +75,10 @@ getJasmineRequireObj().base = function(j$, jasmineGlobal) {
     return obj.nodeType > 0;
   };
 
+  j$.isPromise = function(obj) {
+    return (typeof Promise !== 'undefined') && obj.constructor === Promise;
```

Comment:
I tend to avoid basing logic on `obj.constructor.name` since it can break in production when they're modified by build systems. In a production build using uglify I tested, bluebird's constructor name ended up `"o"`. Figured the global Promise would have more consistent behavior. What do you think?

I did consider being able to set a Promise library, but went for the simplest implementation to start off. I'd be happy to add that configuration option, though.